.github/workflows/pull.yml

```diff
@@ -91,8 +91,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-        with:
```

Comment:
The `codeowners` workflow kept failing at this step: https://github.com/AxtonGrams/terraform-provider-wiz/actions/runs/7106204119/job/19345102741#step:3:53

I think it's because it's attempting to checkout the branch `reports` on the `AxtonGrams/terraform-provider-wiz` repo, and not on the fork. I can look into bringing it back and seeing if I can have it checkout the fork instead of the base repo.